packages/upgrade/src/actions/install.ts

```diff
@@ -123,6 +123,8 @@ async function runInstallCommand(
 	const cwd = fileURLToPath(ctx.cwd);
 	if (ctx.packageManager === 'yarn') await ensureYarnLock({ cwd });
 
+	const installCmd = ctx.packageManager === 'yarn' ? 'add' : 'install'; 
```

Comment:
```suggestion
	const installCmd = ctx.packageManager === 'yarn' || ctx.packageManager === 'pnpm' ? 'add' : 'install'; 
```

While we're at it, we could also perhaps to the same for pnpm. pnpm works with `install` just fine too, but `add` is usually what they recommend instead.